workflows/raredisease.nf

```diff
@@ -215,12 +217,22 @@ workflow RAREDISEASE {
         ch_versions = ch_versions.mix(ch_sv_annotate.versions)
     }
 
-    // STEP 2.1: MT CALLING
+    // STEP 2.1: PREPARING MT ALIGNMENT
+    PREPARE_MT_ALIGNMENT ( ch_mapped.bam_bai )
```

Comment:
Consider renaming the MT subworkflow and split it up into several subworkflows. It does quite a bit more than alignment